packages/backend/tsconfig.json

```
@@ -44,7 +44,13 @@
 
     /* Module Resolution Options */
     "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
-    // "baseUrl": "./" /* Base directory to resolve non-absolute module names. */,
+    "baseUrl": "./" /* Base directory to resolve non-absolute module names. */,
+    "paths": {
+      "@util/*": ["src/util/*"],
+      "@_types/*": ["src/_types/*"],
```

Just a minor but I would like to put the types under `types`

```suggestion
      "@types/*": ["src/types/*"],
```

---

As agreed , we change it to
`@internalTypes/*: ["src/types/*]`

---

Done